prow/plugins/plugin-config-documented.yaml

```diff
@@ -698,7 +698,11 @@ triggers:
     # January 2020.
     trusted_org: ' '
 welcome:
-  - # MessageTemplate is the welcome message template to post on new-contributor PRs
+  - # Post welcome message in all cases, even if PR author is not an existing
+    # contributor or part of the organization
+    everyone: true
```

Comment:
Should this setting be scopeable to global/org/repo?

Reply:
don't you normally manage that by providing a prow plugincfg stanza at the appropriate repo level to override a higher level config?

e.g.

https://github.com/openshift/release/blob/f9cfbaa0027dbfbc75404e32ee16b87f3f5e632b/core-services/prow/02_config/openshift/_pluginconfig.yaml#L1

vs

https://github.com/openshift/release/blob/f9cfbaa0027dbfbc75404e32ee16b87f3f5e632b/core-services/prow/02_config/openshift/ci-search/_pluginconfig.yaml#L1

?

Reply:
Actually that's a really good point, because we do have that kind of fallback mechanism in the plugin itself (first repo config, then org config, then the default) but the ``welcome`` plugin configuration is not being included in the [mergeFrom function](https://github.com/kubernetes/test-infra/blob/f0c7fc8c8a6844180f2580e97825014dbde5838a/prow/plugins/config.go#L1879), making it only applicable in the global configuration (for example see [this](https://github.com/kubernetes/test-infra/pull/25173) )
Let me fix it first, it should be easy enough

Reply:
Done!